UPGRADE.md

```diff
@@ -1,3 +1,16 @@
+## General upgrade instructions
+
+On our internal infrastructure we user deployphp/deployer project
+for deployments/upgrades. Here is example of configuration for publisher: https://gist.github.com/ahilles107/b6dd85f67282a41d81f273fe12344b0e
```

Comment:
`project for deployments/upgrades. Here is an example of the configuration for the Superdesk Publisher:`